pkg/vsphere/toolbox/hgfs/server.go

```diff
@@ -228,6 +236,57 @@ func (s *Server) GetattrV2(p *Packet) (interface{}, error) {
 	return res, nil
 }
 
+// SetattrV2 handles OpSetattrV2 requests
+func (s *Server) SetattrV2(p *Packet) (interface{}, error) {
+	res := &ReplySetattrV2{}
+
+	req := new(RequestSetattrV2)
+	err := UnmarshalBinary(p.Payload, req)
+	if err != nil {
+		return nil, err
+	}
+
+	name := req.FileName.Path()
+
+	uid := -1
+	if req.Attr.Mask&AttrValidUserID == AttrValidUserID {
```

Comment:
This will need normalizing in some manner - we shouldn't be writing these files with the client side uid/gid (which is what I'd expect to see in the tar).
If the docker client doesn't do translation when constructing the tar archive then we need to settle on what to use, which pretty much has to be 0 or the currently defined container USER.